bin/check-package-version

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+
+var exec = require( 'child_process' ).exec;
+var packageJsonPath = require( '../' + process.argv[2] );
```

Comment:
It does not, it contains the package.json. Resolved here:  https://github.com/Automattic/wp-calypso/commit/7ab6fef899288773c06beba947fc327113e20b31